server/utils/config.js

```diff
@@ -0,0 +1,12 @@
+require('dotenv').config()
+
+const MONGODB_URI = process.env.NODE_ENV === 'test'
+  ? process.env.TEST_MONGODB_URI
+  : process.env.MONGODB_URI
+
+const PORT = process.env.PORT
```

Comment:
Remember that your `.env` file is not shared. There whatever you may have set as `PORT` will not be available to me unless I create my own `.env` file and define the `PORT` on my local instance. I don't think perhaps, it might  be a good idea to define a default when `process.env.PORT` is not available.
```javascript
const PORT = process.env.PORT || 5000;
```

What do you think?